sktime/classification/interval_based/vector/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
```

Comment:
Quick question about the module, is `sktime/classification/interval_based/vector/` meant to be a public module for users? I feel like the folder structure becomes a bit deep.

Reply:
Not really. Its mostly just for the interval forests (TSF/DrCIF/CIF/STSF) and its not a time series classifier so it does not need to be public. I just had no idea where to put it really. It currently isent used by default, because it is quite a bit slower than the sklearn tree and i need to figure out why.

I will put it where you think is best.

Reply:
Alright, in that case, I'd just make it private (by convention) using the leading underscore `_vector`. 

As far as I know, the key bits of the scikit-learn tree are written in Cython.

Reply:
Would I have to change the init at all or just the package name?

Yeah i expected it to be, but the differences were quite large! I will take another look.

Reply:
Not that keen on the directory name vector myself, its not informative. Perhaps the class ContinuousIntervalTree should just be within a file, _tsf would be my choice